servant-client/test/Servant/ClientSpec.hs

```diff
@@ -358,7 +359,7 @@ wrappedApiSpec = describe "error status codes" $ do
         test (WrappedApi api, desc) =
           it desc $ bracket (startWaiApp $ serveW api) endWaiApp $ \(_, baseUrl) -> do
             let getResponse :: SCR.ClientM ()
-                getResponse = client api
+                getResponse = client (Proxy :: Proxy ClientM) api
```

Comment:
I'd define `proxyClientM :: Proxy ClientM` on top-level and use it across the spec module.